diff --git a/res/layout/search_container_workspace.xml b/res/layout/search_container_workspace.xml
index 1c617b174..1fd172139 100644
--- a/res/layout/search_container_workspace.xml
+++ b/res/layout/search_container_workspace.xml
@@ -21,10 +21,4 @@
         android:layout_height="0dp"
         android:id="@id/search_container_workspace"
         android:padding="0dp" >
-
-    <fragment
-        android:name="com.android.launcher3.qsb.QsbContainerView$QsbFragment"
-        android:layout_width="match_parent"
-        android:tag="qsb_view"
-        android:layout_height="match_parent"/>
-</com.android.launcher3.qsb.QsbContainerView>
\ No newline at end of file
+</com.android.launcher3.qsb.QsbContainerView>
diff --git a/src/com/android/launcher3/Workspace.java b/src/com/android/launcher3/Workspace.java
index 1441e0b93..2f7eb29f5 100644
--- a/src/com/android/launcher3/Workspace.java
+++ b/src/com/android/launcher3/Workspace.java
@@ -510,12 +510,6 @@ public class Workspace extends PagedView<WorkspacePageIndicator>
             qsb = LayoutInflater.from(getContext())
                     .inflate(R.layout.search_container_workspace, firstPage, false);
         }
-
-        CellLayout.LayoutParams lp = new CellLayout.LayoutParams(0, 0, firstPage.getCountX(), 1);
-        lp.canReorder = false;
-        if (!firstPage.addViewToCellLayout(qsb, 0, R.id.search_container_workspace, lp, true)) {
-            Log.e(TAG, "Failed to add to item at (0, 0) to CellLayout");
-        }
     }
 
     public void removeAllWorkspaceScreens() {
